eng/versioning/version_client.txt

```diff
@@ -67,7 +67,7 @@ com.azure:azure-aot-graalvm-support-netty;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-aot-graalvm-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-aot-graalvm-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-code-customization-parent;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-communication-callautomation;1.2.2;1.3.0-beta.1
+com.azure:azure-communication-callautomation;1.2.2;1.3.0-beta.1,1.3.0-beta.2
```

Comment:
I think here we mention the dependency-version;current-version, in our case now the dependancy-version will be 1.3.0-beta.1 and the current version will be 1.3.0-beta.2 

ex. 1.3.0-beta.1,1.3.0-beta.2

Reply:
Will change thanks!

Reply:
Fixed!